extensions/amp-accordion/1.0/accordion.js

```diff
@@ -72,8 +81,73 @@ export function Accordion({
     });
   }, [expandSingleSection]);
 
+  useImperativeHandle(
+    ref,
+    () => {
```

Comment:
The `useImperativeHandle` result should be defined in our `.type.js`. Just a reminder. I realize this is an early draft.

Reply:
Thanks Dima, updated in latest revision!